apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestDatabaseFixture.cs

```diff
@@ -75,60 +84,122 @@ public class TestDatabaseFixture : IDisposable {
+                    new[] {typeTable});
+
+            // Poll until the returned long-running operation is complete
+            await response.PollUntilCompletedAsync();
+        }
+
+        private async Task CreateTypeTableAsync()
+        {
+            // Create client
+            DatabaseAdminClient databaseAdminClient = await DatabaseAdminClient.CreateAsync();
+
+            var typeTable = @"CREATE TABLE T ( 
+                  K                   STRING(MAX) NOT NULL,
+                  BoolValue           BOOL,
+                  Int64Value          INT64,
+                  Float64Value        FLOAT64,
+                  StringValue         STRING(MAX),
+                  BytesValue          BYTES(MAX),
+                  TimestampValue      TIMESTAMP,
+                  DateValue           DATE,
+                  BoolArrayValue      ARRAY<BOOL>,
+                  Int64ArrayValue     ARRAY<INT64>,
+                  Float64ArrayValue   ARRAY<FLOAT64>,
+                  StringArrayValue    ARRAY<STRING(MAX)>,
+                  BytesArrayValue     ARRAY<BYTES(MAX)>,
+                  TimestampArrayValue ARRAY<TIMESTAMP>,
+                  DateArrayValue      ARRAY<DATE>,
+                ) PRIMARY KEY (K)";
+            // Make the request
+            var response =
+                await databaseAdminClient.UpdateDatabaseDdlAsync(new DatabaseName(TestProjectName,
+                        TestInstanceName,
+                        DatabaseName),
+                    new[] {typeTable});
+
+            // Poll until the returned long-running operation is complete
+            await response.PollUntilCompletedAsync();
+        }
+
         private async Task CreateTableAsync()
         {
             // Create client
             DatabaseAdminClient databaseAdminClient = await DatabaseAdminClient.CreateAsync();
 
-            string createTableStatement = @"CREATE TABLE " + _testTable +" ("
-                                          + "  Key                STRING(MAX) NOT NULL,"
-                                          + "  StringValue        STRING(MAX),"
-                                          + ") PRIMARY KEY (Key)";
+            string createTableStatement = $@"CREATE TABLE {TestTable} (
+                                            Key                STRING(MAX) NOT NULL,
+                                            StringValue        STRING(MAX),
+                                          ) PRIMARY KEY (Key)";
             string index1 = "CREATE INDEX TestTableByValue ON TestTable(StringValue)";
             string index2 = "CREATE INDEX TestTableByValueDesc ON TestTable(StringValue DESC)";
 
             // Make the request
             var response =
-                await databaseAdminClient.UpdateDatabaseDdlAsync(new DatabaseName(TestProjectName, TestInstanceName, DatabaseName),
-                new[] { createTableStatement });
+                await databaseAdminClient.UpdateDatabaseDdlAsync(
```

Comment:
possibly the 2 indices could be created in parallel.  However, that would result in multiple sessions, which is probably ok -- but I often like to double check that only 1 session is created for certain tests.